src/test/shell/bazel/remote/remote_execution_http_test.sh

```diff
@@ -76,6 +76,10 @@ EOF
     || fail "Failed to build //a:test without remote cache"
   cp -f bazel-bin/a/test ${TEST_TMPDIR}/test_expected
 
+  bazel clean --expunge
```

Comment:
remove the --expunge please. it's not necessary and kills the Bazel server thus requiring a restart on the next invocation